cpp/include/cudf/lists/detail/scatter.cuh

```diff
@@ -117,8 +117,8 @@ std::unique_ptr<column> scatter_impl(
 
   auto list_size_begin = thrust::make_transform_iterator(
     target_vector.begin(), [] __device__(unbound_list_view l) { return l.size(); });
-  auto offsets_column = cudf::strings::detail::make_offsets_child_column(
-    list_size_begin, list_size_begin + target.size(), stream, mr);
+  auto offsets_column = std::get<0>(cudf::detail::make_offsets_child_column(
```

Comment:
Question: Sometimes I have seen we use a structured binding with a throwaway like:
```suggestion
  auto [offsets_column, _] = cudf::detail::make_offsets_child_column(
```
I think we use this structured binding style in quite a few places but am uncertain about whether there is a strong argument for/against this pattern. Are you aware of an argument for/against?

Reply:
I saw a review recently that suggested using `get<n>` instead of the `_` in places where the parameter can indeed be discarded. My worry has always been that I would get an `unused variable warning` for `_` even though I've not seen that occur yet.
Also, I agree that it seems cleaner to me as in this review comment: https://github.com/rapidsai/cudf/pull/12239#discussion_r1050099794

Reply:
Sounds good!